PelumiAlesh/Banka
# Ft user pages 164803770

What does this PR do?

Fix Login Button

Description of task to be completed

Button links to Profile

How should this be manually tested

after cloning, cd into login.html and check button

Any Background context you want to provide?

What are the relevant Pivot Tracker stories

164803770
